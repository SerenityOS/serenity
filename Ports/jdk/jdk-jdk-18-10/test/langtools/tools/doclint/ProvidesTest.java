/*
 * @test /nodynamiccopyright/
 * @bug 8160196
 * @summary Module summary page should display information based on "api" or "detail" mode.
 * @modules jdk.javadoc/jdk.javadoc.internal.doclint
 * @build DocLintTester
 * @run main DocLintTester -ref ProvidesTest.out ProvidesTest.java
 */

/**
 * Invalid use of provides in class documentation.
 *
 * @provides NotFound
 */
public class ProvidesTest {
    /**
     * Invalid use of provides in field documentation
     *
     * @provides NotFound Test description.
     */
    public int invalid_param;

    /**
     * Invalid use of provides in method documentation
     *
     * @provides NotFound Test description.
     */
    public class InvalidParam { }
}
