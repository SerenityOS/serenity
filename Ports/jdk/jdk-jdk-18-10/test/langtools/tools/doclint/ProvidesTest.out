ProvidesTest.java:13: error: invalid use of @provides
 * @provides NotFound
   ^
ProvidesTest.java:13: error: service-type not found
 * @provides NotFound
   ^
ProvidesTest.java:13: error: reference not found
 * @provides NotFound
             ^
ProvidesTest.java:19: error: invalid use of @provides
     * @provides NotFound Test description.
       ^
ProvidesTest.java:19: error: service-type not found
     * @provides NotFound Test description.
       ^
ProvidesTest.java:19: error: reference not found
     * @provides NotFound Test description.
                 ^
ProvidesTest.java:26: error: invalid use of @provides
     * @provides NotFound Test description.
       ^
ProvidesTest.java:26: error: service-type not found
     * @provides NotFound Test description.
       ^
ProvidesTest.java:26: error: reference not found
     * @provides NotFound Test description.
                 ^
9 errors
